app/src/main/java/com/nytimes/android/sample/SampleRoomStore.kt

```diff
@@ -0,0 +1,56 @@
+package com.nytimes.android.sample
+
+import android.arch.persistence.room.Dao
+import android.arch.persistence.room.Database
+import android.arch.persistence.room.Entity
+import android.arch.persistence.room.Insert
+import android.arch.persistence.room.PrimaryKey
+import android.arch.persistence.room.Query
+import android.arch.persistence.room.Room
+import android.arch.persistence.room.RoomDatabase
+import com.nytimes.android.external.store3.base.Fetcher
+import com.nytimes.android.external.store3.base.impl.room.StoreRoom
+import com.nytimes.android.external.store3.base.room.RoomPersister
+import io.reactivex.Flowable
+import io.reactivex.Observable
+import io.reactivex.Single
+
+@Entity
+data class User(
+        @PrimaryKey(autoGenerate = true)
+        var uid: Int = 0,
+        val name: String)
+
+@Dao
+interface UserDao {
+    @Query("SELECT name FROM user")
+    fun loadAll(): Flowable<List<String>>
+
+    @Insert
+    fun insertAll(user: User)
+
+}
+
+@Database(entities = arrayOf(User::class), version = 1)
+abstract class AppDatabase : RoomDatabase() {
+    abstract fun userDao(): UserDao
+}
+
+val db = Room.databaseBuilder(SampleApp.appContext!!, AppDatabase::class.java, "db").build()
```

Comment:
Reimplement to avoid using the `!!` operator? 
Can be as simple as creating a builder that just takes a context.